src/pages/pricing-plans/PricingPlans.vue

```diff
@@ -0,0 +1,91 @@
+<template>
+  <div class="text-textPrimary">
+    <h1 class="text-[32px] md:text-5xl leading-[110%] md:leading-[56px] font-bold">
+      Choose your plan
+    </h1>
+    <div class="py-4 text-lg leading-[26px]">
+      If you need more info about our pricing, please check <span class="text-primary underline">Pricing Guidelines</span>.
+    </div>
+    <div class="flex flex-col p-4 bg-backgroundSecondary">
+      <div class="flex justify-center">
+        <div class="space-x-[1px] p-[2px] border rounded">
+          <va-button 
+            v-for="option in ['Monthly', 'Annual']"
+            class="w-[85.5px]"
```

Comment:
you can standard tailwind classes or at least rounder numbers.

Maybe also use rem? (not sure if that's correct idea)